runtime/runtime/src/ext.rs

```diff
@@ -76,20 +76,30 @@ impl<'a> RuntimeExt<'a> {
 impl<'a> External for RuntimeExt<'a> {
     fn storage_set(&mut self, key: &[u8], value: &[u8]) -> Result<Option<Vec<u8>>, ExternalError> {
         let storage_key = self.create_storage_key(key);
-        let evicted = self.trie_update.get(&storage_key).map(DBValue::into_vec);
+        let evicted = self
+            .trie_update
+            .get(&storage_key)
+            .map_err(|_| ExternalError::StorageError)?
```

Comment:
We should probably pipe specific error in `External` instead of dropping it, e.g. `.map_err(|err| ExternalError::StorageError(err))?` Or we could implement `into` for `StorageError` and then just do `?` without `map_err`.